recipes/sfml/3.0.0/conanfile.py

```diff
@@ -0,0 +1,270 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, rmdir, copy
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+from conan.tools.build import check_min_cppstd
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class SfmlConan(ConanFile):
+    name = "sfml"
+    description = "Simple and Fast Multimedia Library."
+    license = "Zlib"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.sfml-dev.org"
+    topics = ("multimedia", "games", "graphics", "audio")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "window": [True, False],
+        "graphics": [True, False],
+        "network": [True, False],
+        "audio": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "window": True,
+        "graphics": True,
+        "network": True,
+        "audio": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def build_requirements(self):
+        self.tool_requires('cmake/[>=3.24]')
```

Comment:
3.24 is only required for Windows. All other OSes only need 3.22.

Reply:
Are we really bothered by 2 minor version releases that are 8 months apart in release, which there are like 5 known minor distributions that have 3.22.0 but do not have 3.24.0?

Reply:
Haha I love the sassy commit message. It's between you and the Conan Center maintainers to figure out whether this level of pedantry is worthwhile. I'm indifferent. I'm just here as an SFML maintainer to make sure you have all the best and up-to-date information on the library.

Reply:
Appreciate it. Didn't even realize you were a / the maintainer for SFML to be honest.

Personally I (unsurprisingly) am of the opinion that it should just be the higher number in this case. If it were some major difference, or one requirement is a relatively recent version (i.e. a few months old), sure there can be a difference between the systems.

Ultimately though I'm not going to die on that hill, which is why I changed it. Couldn't help myself about the commit message though 😉